source/codegen/metadata/nixnet/functions.py

```diff
@@ -374,6 +374,7 @@
     },
     'DbCloseDatabase': {
         'cname': 'nxdbCloseDatabase',
+        'custom_close_method': True,
```

Comment:
Note: as long as you are generating functions from scrapigen, you will have to back-port these changes to the extraction tool.

Reply:
ALSO: I don't see `custom_close_method` being used. Why was that added?

Reply:
We will be back-porting any changes from here to scrapigen. That will be done once this PR is finalized. 

`custom_close_method` tag is used in `service_helpers.mako`. We can mark only one function as `close_function` in `config.py`. This tag is used for tagging any other APIs which are close methods to generate `remove_session` call. This particular close function is used by `DbOpenDatabase`.